astro.config.mjs

```diff
@@ -20,7 +20,7 @@ export default defineConfig({
   integrations: [
     vue(),
     tailwind({
-      config: { applyBaseStyles: false },
+      applyBaseStyles: false,
```

Comment:
[Astro's Tailwind integration made this change in v4.0](https://github.com/withastro/astro/blob/main/packages/integrations/tailwind/CHANGELOG.md#400)